src/app/components/popup-host/popup-host.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="app-popup-content-wrapper" [ngClass]="{hidden: !template()}">
```

Comment:
Я всегда рекомендую подход с биндингом классов, он читается как то попроще и биндинг более нативная история, Ангуляру чуть проще ее оптимизировать и мигрировать на новые инструменты
```suggestion
<div class="app-popup-content-wrapper" [class.hidden]="!template()">
```